openedx/core/djangoapps/ace_common/settings/common.py

```diff
@@ -10,6 +10,9 @@ def plugin_settings(settings):
     settings.ACE_CHANNEL_SAILTHRU_API_KEY = None
     settings.ACE_CHANNEL_SAILTHRU_API_SECRET = None
 
+    email_getter = 'openedx.core.djangoapps.site_configuration.helpers:get_email_from_address'
+    settings.ACE_CHANNEL_DJANGO_FROM_ADDRESS_GETTER = email_getter
```

Comment:
Hooray!  Looks like you're tackling an issue that our White Label team is needing to address: [Customize From/Sender Emails for White Label in Sailthru (WL-1388)](https://openedx.atlassian.net/browse/WL-1388).
FYI @bill-filler

Reply:
Glad I'm helping!

Reply:
@OmarIthawi @nasthagiri 
excellent :)